client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize.jsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card/compact';
+import Main from 'components/main';
+import Notice from 'components/notice';
+import upgradesActions from 'lib/upgrades/actions';
+import DomainConnectAuthorizeDescription from './domain-connect-authorize-description';
+import DomainConnectAuthorizeConflicts from './domain-connect-authorize-conflicts';
+import DomainConnectAuthorizeFooter from './domain-connect-authorize-footer';
+import { actionType, noticeType } from './constants';
+
+class DomainConnectAuthorize extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			action: null,
+			dnsTemplateConflictsRetrieved: false,
+			noticeType: null
+		};
+	}
+
+	componentDidMount() {
+		const { providerId, params } = this.props,
+			{ domain } = params;
+
+		upgradesActions.getDnsTemplateConflicts( domain, providerId, params, ( error, data ) => {
+			if ( ! error ) {
+				this.setState( {
+					action: actionType.READY_TO_SUBMIT,
+					dnsTemplateConflicts: data.records,
+					dnsTemplateError: false
+				} );
+			} else {
+				const errorMessage = error.message ||
+					translate( 'We aren\'t able to set up the service with the information given. Please check ' +
+						'with your service provider to make sure they provided all the correct data.' );
+
+				this.setState( {
+					action: actionType.CLOSE,
+					dnsTemplateError: true,
+					noticeType: noticeType.ERROR,
+					noticeMessage: errorMessage
+				} );
+			}
+
+			this.setState( {
+				dnsTemplateConflictsRetrieved: true
+			} );
+		} );
+	}
+
+	handleClickConfirm = () => {
+		const { providerId, params } = this.props,
+			{ domain } = params;
+
+		this.setState( {
+			action: actionType.SUBMITTING,
+			notice: null
```

Comment:
Yes. Another good catch! I changed this from `notice` to `noticeType` at one point because it made more sense to me and I totally missed this one.